common/components/utils/glyphs.js

```diff
@@ -43,4 +43,26 @@ export function Glyph(style: string, size: ?string): string {
   return style + (size ? " " + size : "");
 }
 
+//to use GlyphOption you must also import GlyphStyles, GlyphSizes, from glyphs.js
+export function GlyphOption(style: string, options: object) {
```

Comment:
This solution works.  There's a bit more code than I'd prefer.  If I had to make it work with a minimum of code, what I would do is change the Glyph function to accept a variable number of style add-ons, like
export function Glyph(style: string, ...args)

So we can just tack on extra style parameters as needed.  Ex:
Glyph(GlyphStyles.Pushpin, GlyphSizes.X2, GlyphWidth.Fixed)

With the added bonus that we wouldn't have to change any existing calls to Glyph()

Reply:
I just pushed something that should do this. Array.join didn't take to me using " " as a separator (empty strings are ignored) so I just modified GlyphSizes and GlyphWidth to include the spaces we need for the class declarations.